ledger/participant-integration-api/src/main/resources/db/migration/postgres/V51__drop_configuration_from_parameters_table.sha256

```diff
@@ -0,0 +1 @@
+6326123dd12a9d3652369df4ff68e7479716307b95960211aefa08fc8a6ce057
```

Comment:
Note to self: this file will have to be renamed to V101 or merged with the V100 migration after https://github.com/digital-asset/daml/pull/9481 is merged.